plugins/node/opentelemetry-plugin-pg-pool/test/pg-pool.test.ts

```diff
@@ -113,14 +114,15 @@ describe('pg-pool@2.x', () => {
     if (testPostgresLocally) {
       testUtils.startDocker('postgres');
     }
-    done();
   });
 
   after(done => {
     if (testPostgresLocally) {
       testUtils.cleanUpDocker('postgres');
     }
+    console.log('ending pool');
```

Comment:
is it intentional?

Reply:
leftovers from intensive testing yesterday